docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+scriptFilePath=$SCRIPT_FILE_PATH
+if [ -z "$scriptFilePath" ]; then
+  scriptFilePath="/replacements.sed"
+  #DEFAULT
+fi
+
+inotifywait -mr "$SOURCE_ROOT" -e moved_to -e create --format '%w%f' |
+  while read -r fullPath; do
+    ending="${rest: -1}"
+    if [ -d "$fullPath" ] && [ "$ending" != "/" ]; then
+      fullPath="$fullPath/"
+    fi
+
+    originalPath=${fullPath#"$SOURCE_ROOT"}
+    replacedPath=$(echo "$originalPath" | sed -r -f $scriptFilePath)
+    if [[ "$originalPath" != "$replacedPath" ]]; then
+      echo "mapped $originalPath to $replacedPath"
```

Comment:
do not print to stout but to stderr for logging, anyways i think it is a bit too verbose in the future to print out everything

Reply:
below logging is enough, remove this line